uint32 robot_x
uint32 robot_y
---
nav_msgs/GridCells map